soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -64,6 +64,10 @@ config IMX_USDHC
 	default y if (HAS_MCUX_USDHC1 || HAS_MCUX_USDHC2)
 	depends on SDHC
 
+config ADC_MCUX_ETC
+	default y if HAS_MCUX_ADC_ETC
```

Comment:
Can you remove the `default y` here? Since there is no upstream driver present for the ADC external trigger, this default will cause the ADC EXT driver to be compiled in, and rely on the linker to strip the unused functions this HAL driver contains. I'd suggest removing this section in the `Kconfig.defconfig.series` entirely, so that the driver is only enabled explicitly by your application.

Reply:
Done.